base/applications/rapps/installed.cpp

```diff
@@ -10,12 +10,11 @@
 
 #include "installed.h"
 
-#include "gui.h"
 #include "misc.h"
 
-BOOL GetApplicationString(HKEY hKey, LPCWSTR lpKeyName, ATL::CStringW& String)
+BOOL INSTALLED_INFO::GetApplicationString(LPCWSTR lpKeyName, ATL::CStringW& String)
```

Comment:
This could be merged with `::GetApplicationString()` since it was only used in ATL::CStringW form...